platform.md

```diff
@@ -955,6 +1042,11 @@ Where:
 id = "<buildpack ID>"
 version = "<buildpack version>"
 optional = false
+
+[[order-ext]]
+[[order-ext.group]]
```

Comment:
1. Should this be extensions since it seems like we are expanding the term everywhere else?
2. Do we prefer `order-extensions` or `extensions-order`? To me, `extensions-order` reads better.